src/helpers/math.cpp

```diff
@@ -619,4 +619,24 @@ QAngle math::AngleNormalize(QAngle angle)
 		angle.yaw += 360;
 
 	return angle;
+}
+
+float math::Magnitude(Vector a)
```

Comment:
The Vector class has already method to get magnitude, I think its called Lenght or something like that.